src/main/java/org/sonarsource/sonarlint/ls/AnalysisManager.java

```diff
@@ -301,6 +303,12 @@ private void analyze(URI fileUri, boolean shouldFetchServerIssues) {
       LOG.debug("Skipping analysis of Java file '{}' because SonarLint was unable to query project configuration (classpath, source level, ...)", fileUri);
       return;
     }
+    String filePath = fileUri.getPath();
+    Boolean isIgnored = filesIgnoredByScmCache.computeIfAbsent(filePath, s -> client.isIgnoredByScm(s).join());
+    if (isIgnored) {
+      LOG.debug("Skip analysis for SCM ignored file: " + fileUri);
```

Comment:
```suggestion
      LOG.debug("Skip analysis for SCM ignored file: '{}'", fileUri);
```
(very minor optimization)